docs/appconfig/schema-definitions-dependencyendpoint-properties-apipaths-items.md

```diff
@@ -0,0 +1,16 @@
+# Untitled string in AppConfig Schema
```

Comment:
Still looks like an unrelated API docs change, probably fixed now that #888 went in... I think you can delete this file and if you have revised then "make pre-push" should not add it again...